src/routers/api.ts

```diff
@@ -151,7 +149,7 @@ router.post('/messages', async (req, res, next) => {
   }
 });
 
-router.head('/blobs/*', async (req, res, next) => {
+router.head('/blobs/*', async (req: Request, res, next) => {
```

Comment:
Are these needed?

Reply:
I found this was the trick to force the TS compiler to pick up the `config.d.ts` extensions